inst/@sym/cell2sym.m

```diff
@@ -0,0 +1,56 @@
+%% Copyright (C) 2016 Lagu
+%%
+%% This program is free software; you can redistribute it and/or
+%% modify it under the terms of the GNU General Public
+%% License as published by the Free Software Foundation;
+%% either version 3, or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied
+%% warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+%% PURPOSE.  See the GNU General Public License for more
+%% details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.  If not,
+%% see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @defmethod @@sym cell2sym (@var{x})
+%% Convert cell array to symbolic array.
+%%
+%% Examples:
+%% @example
+%% @group
+%% syms x y
+%% cell2sym({x, y})
+%%   @result{} ans = (sym) [x  y]  (1×2 matrix)
+%% @end group
+%% @end example
```

Comment:
Can you add an example like `cell2sym({'x', 2; pi 'y'})'?  Or does that not work yet?  It would be an `xtest` if its not working yet.